build.gradle

```
@@ -4,27 +4,21 @@ plugins {
     id 'java'
 }
 
-group = 'camp.nextstep.edu'
-version = '0.0.1-SNAPSHOT'
-sourceCompatibility = '1.8'
+allprojects {
 
-repositories {
-    mavenCentral()
-}
-
-dependencies {
-    implementation 'org.springframework.boot:spring-boot-starter-web'
-    implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
-    implementation 'org.springframework.boot:spring-boot-starter-validation'
+    group = 'camp.nextstep.edu'
+    version = '0.0.1-SNAPSHOT'
+    sourceCompatibility = '1.8'
 
-    implementation 'org.springframework.boot:spring-boot-starter-actuator'
-    implementation 'org.springframework.boot:spring-boot-starter-jdbc'
+    repositories {
+        mavenCentral()
+    }
 
-    implementation 'org.flywaydb:flyway-core'
-    runtimeOnly 'com.h2database:h2'
-    testImplementation 'org.springframework.boot:spring-boot-starter-test'
-}
+    apply {
+        plugin("io.spring.dependency-management")
+    }
```

plugin이란 함수는 없는 걸로 알고 있습니다. (혹시나.. 제가 모르는 새로운 방법이 나온거라면 알려주시면 감사하겠습니다.. ㅎㅎ)
제가 아는 바로는 플러그인을 적용하려면 아래와 같이 사용해야 합니다

```suggestion
    apply plugin: 'io.spring.dependency-management'
```

---

kotlinDsl 사용하다가 저도 모르게... ㅠㅠ